sklearn/tests/test_multioutput.py

```diff
@@ -601,3 +601,24 @@ def fit(self, X, y, **fit_params):
 
     for est in model.estimators_:
         assert est.sample_weight_ is weight
+
+
+def test_classifier_chain_tuple_order():
```

Comment:
You can parametrize this function with the type and assert the result of predictions only once.

```suggestion

@pytest.mark.parametrize(
    'y_type', [list, np.array, tuple], ids=['list', 'ndarray', 'tuple']
)
def test_classifier_chain_tuple_order(y_type):
```

Reply:
did not know this, thanks